components/menu-breadcrumbs-sd.tpl

```diff
@@ -3,9 +3,8 @@
 
 <ul class="menu menu-item-list">
   {%- for listItem in breadcrumbsObj.itemListElement %}
-    {%- assign pageUrl = page.url | remove_first: "/" -%}
     <span class="menu-separator">/</span>
-    <li class="selected menu-item {% if pageUrl == listItem.item.id %} current{% endif %}">
+    <li class="selected menu-item {% if forloop.last %}current{% endif %}">
```

Comment:
Here the `selected` class does provide a `font-weight` value, but it is the same as the `body` element's value. We can remove this.